### ISSUE TITLE
argon2 bottle sha256 mismatch

### ISSUE THREAD

**Issue by fluff** [Homebrew/homebrew-core#41576]
- [x] are reporting a bug others will be able to reproduce and not asking a question or requesting software. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh. To get software added or changed in Homebrew please file a [Pull Request](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)
- [x] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, Homebrew/homebrew-core formula (not cask) on macOS? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a Linux problem please file this issue at https://github.com/Homebrew/linuxbrew-core/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [x] ran `brew update` and can still reproduce the problem?
- [ ] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [ ] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [ ] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:

- What you were trying to do (and why)

`brew upgrade` tries to pull in a new argon2 bottle 

- What happened (include command output)

```
==> Upgrading 1 outdated package:
argon2 20171227 -> 20190702
==> Upgrading argon2 
==> Downloading https://homebrew.bintray.com/bottles/argon2-20190702.mojave.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/92/922dae39eb90c1f47531ad331470a3a23ff9196ef7e380f4f3f172055c38bb0f?__gda__=exp=156
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: da0c88fc168a19ebaf26d4f1289b88ef35bc837412846d5bbd144ed22dee4059
  Actual: 922dae39eb90c1f47531ad331470a3a23ff9196ef7e380f4f3f172055c38bb0f
```

- What you expected to happen

No SHA256 mismatch

- Step-by-step reproduction instructions (by running `brew install` commands)

`brew upgrade`

Note: Looking at the bottle .tar.gz, the file timestamps within the tarball are from May 20th, which doesn't make sense since the paths include "20190702". Is the system clock on the bottle builder machines off by more than a month? 

```
% tar tvzf  /Users/username/Library/Caches/Homebrew/downloads/0abb94956a7eae761b751f5b64c4b28d8c2f886122332f88f635140d4fda25a2--argon2-20190702.mojave.bottle.tar.gz
drwxr-xr-x  0 brew   staff       0 May 20 11:18 argon2/20190702/
-rw-r--r--  0 brew   staff     601 May 20 11:18 argon2/20190702/INSTALL_RECEIPT.json
-rw-r--r--  0 brew   staff   17879 May 20 11:18 argon2/20190702/LICENSE
drwxr-xr-x  0 brew   staff       0 May 20 11:18 argon2/20190702/bin/
-rw-r--r--  0 brew   staff     765 May 20 11:18 argon2/20190702/CHANGELOG.md
drwxr-xr-x  0 brew   staff       0 May 20 11:18 argon2/20190702/.brew/
drwxr-xr-x  0 brew   staff       0 May 20 11:18 argon2/20190702/include/
-rw-r--r--  0 brew   staff   13157 May 20 11:18 argon2/20190702/README.md
drwxr-xr-x  0 brew   staff       0 May 20 11:18 argon2/20190702/lib/
drwxr-xr-x  0 brew   staff       0 May 20 11:18 argon2/20190702/share/
drwxr-xr-x  0 brew   staff       0 May 20 11:18 argon2/20190702/share/doc/
drwxr-xr-x  0 brew   staff       0 May 20 11:18 argon2/20190702/share/doc/argon2/
-rw-r--r--  0 brew   staff  459608 May 20 11:18 argon2/20190702/share/doc/argon2/argon2-specs.pdf
drwxr-xr-x  0 brew   staff       0 May 20 11:18 argon2/20190702/lib/pkgconfig/
-r--r--r--  0 brew   staff   32144 May 20 11:18 argon2/20190702/lib/libargon2.a
lrwxr-xr-x  0 brew   staff       0 May 20 11:18 argon2/20190702/lib/libargon2.dylib -> libargon2.1.dylib
-r--r--r--  0 brew   staff   28604 May 20 11:18 argon2/20190702/lib/libargon2.1.dylib
-r--r--r--  0 brew   staff     332 May 20 11:18 argon2/20190702/lib/pkgconfig/libargon2.pc
-rw-r--r--  0 brew   staff   16767 May 20 11:18 argon2/20190702/include/argon2.h
-rw-r--r--  0 brew   staff     744 May 20 11:18 argon2/20190702/.brew/argon2.rb
-r-xr-xr-x  0 brew   staff   33388 May 20 11:18 argon2/20190702/bin/argon2
```

**Comment by SMillerDev** [comment]
Had the same thing, https://github.com/Homebrew/homebrew-core/pull/41579 should fix it.

**Comment by SMillerDev** [comment]
Fixed by #41557 instead
